apps/aechannel/test/aesc_fsm_SUITE.erl

```diff
@@ -962,6 +985,50 @@ wrong_sig_action(ChannelStuff, Poster, Malicious,
 wrong_sig_action({I, R, _Spec, _Port, Debug}, Poster, Malicious,
                  {FsmFun, FsmFunArg, FsmNewAction, FsmCreatedAction},
                   DetectConflictFun) ->
+    wrong_action({I, R, _Spec, _Port, Debug}, Poster, Malicious,
+                 {FsmFun, FsmFunArg, FsmNewAction, FsmCreatedAction},
+                  DetectConflictFun,
+                  fun(Action, Signer, Debug1) ->
+                      {_, _WrongSignedTx} =
+                          await_signing_request(Action, Signer#{priv =>
+                                                                ?BOGUS_PRIVKEY},
+                                                Debug1)
+                  end,
+                  bad_signature).
+
+wrong_id_action(ChannelStuff, Poster, Malicious,
+                 FsmStuff) ->
+    wrong_id_action(ChannelStuff, Poster, Malicious,
+                  FsmStuff,
+                  fun(Who, DebugL) ->
+                      {ok, _} = receive_from_fsm(conflict, Who, any_msg(), ?TIMEOUT, DebugL)
+                  end).
+
+wrong_id_action({I, R, _Spec, _Port, Debug}, Poster, Malicious,
+                {FsmFun, FsmFunArg, FsmNewAction, FsmCreatedAction},
+                 DetectConflictFun) ->
+    wrong_action({I, R, _Spec, _Port, Debug}, Poster, Malicious,
+                 {FsmFun, FsmFunArg, FsmNewAction, FsmCreatedAction},
+                  DetectConflictFun,
+                  fun(Tag, #{priv := Priv} = Signer, Debug1) ->
+                      Action = fun sign_signing_request/3,
+                      receive {aesc_fsm, Fsm, #{type := sign, tag := Tag, info := Tx0} = Msg} ->
+                              log(Debug1, "await_signing(~p, ~p) <- ~p", [Tag, Fsm, Msg]),
+                              Tx = aesc_test_utils:update_tx(Tx0, channel_id,
+                                                             aec_id:create(channel,
+                                                                          ?BOGUS_PUBKEY)),
+                              log(Debug1, "modified ~p", [Tx]),
+                              SignedTx = aec_test_utils:sign_tx(Tx, [Priv]),
+                              Action(Tag, Signer, SignedTx)
+                      after ?TIMEOUT ->
+                              error(timeout)
+                      end
+                  end,
+                  different_channel_id).
```

Comment:
I tried changing this, that is new error reason introduced by this PR, and test failed as expected. Ok.

```
%%% aesc_fsm_SUITE ==> all_tests.channel_ids.check_incorrect_deposit: FAILED
%%% aesc_fsm_SUITE ==> {timeout,[{aesc_fsm_SUITE,receive_from_fsm,6,
```